modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -2013,7 +2013,7 @@ void ONNXImporter::handleNode(const opencv_onnx::NodeProto& node_proto_)
                 layerParams.set("align_corners", interp_mode == "align_corners");
                 if (layerParams.get<String>("mode") == "linear")
                 {
-                    layerParams.set("mode", interp_mode == "pytorch_half_pixel" ?
+                    layerParams.set("mode", interp_mode == "pytorch_half_pixel" || interp_mode == "half_pixel" ?
```

Comment:
According to ONNX spec `coordinate_transformation_mode` attribute is NOT defined for ["Upsample" operator](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Upsample) (which is deprecated since ONNX v10).

Perhaps we need here some framework-specific code (with proper links on frameworks documentation for related changes)